src/styles/globals.css

```diff
@@ -123,3 +123,31 @@ body {
 .ais-SearchBox input {
   @apply h-full w-full rounded-xl border-2 border-primary/30 bg-transparent p-4 text-3xl outline-none placeholder:text-base-content focus:ring-2 focus:ring-primary;
 }
+
```

Comment:
@abhinav-m22  we are using Yarn package manager for installing dependencies and not npm package manager for installation.
Please remove package-lock.json file and install the dependencies using yarn.
Commit those changes.